apps/chat/src/store/models/models.reducers.ts

```diff
@@ -253,6 +259,13 @@ const selectIsModelsLoaded = createSelector([rootSelector], (state) => {
   return state.status === UploadStatus.LOADED;
 });
 
+const selectIsInstalledModelsInitialized = createSelector(
+  [rootSelector],
+  (state) => {
+    return state.installedModelsStatus !== UploadStatus.UNINITIALIZED;
```

Comment:
need to exclude `LOADING` also
```suggestion
    return state.installedModelsStatus === UploadStatus.LOADED;
```

Reply:
so we will have global loading each time when we open settings. I don't think that is good idea